gnomad/sample_qc/relatedness.py

```diff
@@ -1298,3 +1300,415 @@ def _get_alt_count(locus, gt, is_female):
+    method. Please refer to these sources for more information on the de novo model.
+
+    Neither Kaitlin's de novo caller nor Hail's de novo method provide a clear
+    description on how to calculate for de novo calls for hemizygous genotypes in XY
+    individuals. These equations are included below:
+
+    .. math::
+
+        P_{dn} = \frac{P(DN \mid \text{data})}{P(DN \mid \text{data}) + P(\text{missed het in parent(s)} \mid \text{data})}
+
+    The terms are defined as follows:
+
+    - :math:`P(DN \mid \text{data})` is the probability that the variant is **de novo**, given the observed genotype data.
+
+    - :math:`P(\text{missed het in parent(s)} \mid \text{data})` is the probability that the heterozygous variant was **missed in the parent(s)**.
+
+    Applying Bayesian Theorem to the numerator and denominator yields:
+
+    .. math::
+
+        P_{dn} = \frac{P(\text{data} \mid DN) \, P(DN)}{P(\text{data} \mid DN) \, P(DN) + P(\text{data} \mid \text{missed het in parent(s)}) \, P(\text{missed het in parent(s)})}
+
+    where:
+
+    - :math:`P(\text{data} \mid DN)`: Probability of observing the data under the assumption of a de novo mutation.
+
+      - **Autosomes and PAR regions**:
+
+        .. math::
+
+            P(\text{data} \mid DN) = P(\text{hom_ref in father}) \, P(\text{hom_ref in mother}) \, P(\text{het in proband})
+
+      **Probability of a de novo mutation given the data for hemizygous calls in XY individuals**
+
+      - **X non-PAR regions (XY only)**:
+
+        .. math::
+
+            P(\text{data} \mid DN) = P(\text{hom_ref in mother}) \, P(\text{hom_alt in proband})
+
+      - **Y non-PAR regions (XY only)**:
+
+        .. math::
+
+            P(\text{data} \mid DN) = P(\text{hom_ref in father}) \, P(\text{hom_alt in proband})
+
+    - :math:`P(DN)`: The prior probability of a de novo mutation from literature, defined as:
+
+      .. math::
+
+          P(DN) = \frac{1}{3 \times 10^7}
+
+    - :math:`P(\text{data} \mid \text{missed het in parent(s)})`: Probability of observing the data under the assumption of a missed het in a parent.
+
+      - **Autosomes and PAR regions**:
+
+        .. math::
+
+            P(\text{data} \mid \text{missed het in parents}) = ( P(\text{het in father}) \times P(\text{hom_ref in mother}) + P(\text{hom_ref in father}) \times P(\text{het in mother})) \times P(\text{het in proband})
+
+      - **X non-PAR regions**:
+
+        .. math::
+
+            P(\text{data} \mid \text{missed het in mother}) = (P(\text{het in mother}) + P(\text{hom_alt in mother})) \times P(\text{hom_alt in proband})
+
+      - **Y non-PAR regions**:
+
+        .. math::
+
+            P(\text{data} \mid \text{missed het in father}) = (P(\text{het in father}) + P(\text{hom_alt in father})) \times P(\text{hom_alt in proband})
+
+    - :math:`P(\text{missed het in parent(s)` equals the **probability for at least one heterozygous parent**:
+
```

Comment:
adding this here as well since you mentioned you are going to fix the formatting change, and I don't want the rewording to get lost: 
![image](https://github.com/user-attachments/assets/ed512a3f-06c4-40ef-bc94-faca94dc010f)

```
Prior that at least one parent is heterozygous. Depends on alternate allele frequency
```